apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -64,6 +64,14 @@ class VirtualizedDetailView extends Component {
     scrollToRow: 0,
   };
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
not ready for review - I'm working on replacing `componentWillRecieveProps` with the new recommended approach.